app/Agri.Data/SeedData/StaticDataVersion_4.json

```diff
@@ -424016,19 +424016,155 @@
     {
       "Id": 1,
       "Name": "Low (2.3% of body weight per day)",
-	  "Value": "0.023",
+      "Value": "0.023",
       "StaticDataVersionId": 4
     },
     {
       "Id": 2,
-     "Name": "Medium (2.6% of body weight per day)",
-	  "Value": "0.026",
+      "Name": "Medium (2.6% of body weight per day)",
+      "Value": "0.026",
       "StaticDataVersionId": 4
     },
     {
       "Id": 3,
-     "Name": "High (3.0% of body weight per day)",
-	  "Value": "0.03",
+      "Name": "High (3.0% of body weight per day)",
+      "Value": "0.03",
+      "StaticDataVersionId": 4
+    }
+  ],
+  "FeedEfficiencies": [
+    {
+      "Id": 1,
+      "AnimalType": "mature animals",
+      "Nitrogen": "1",
+      "Phosphorous": "1",
+      "Potassium": "1",
+      "StaticDataVersionId": 4
+    },
+    {
+      "Id": 2,
+      "AnimalType": "growing animals",
+      "Nitrogen": "0.97",
+      "Phosphorous": "0.9928",
+      "Potassium": "0.9957",
+      "StaticDataVersionId": 4
+    }
+  ],
+  "FeedConsumptions": [
+    {
+      "Id": 1,
+      "Label": "Low (2.3% of body weight per day)",
+      "Value": "0.023",
+      "StaticDataVersionId": 4
+    },
+    {
+      "Id": 2,
+      "Label": "Medium (2.6% of body weight per day)",
+      "Value": "0.026",
+      "StaticDataVersionId": 4
+    },
+    {
+      "Id": 3,
+      "Label": "High (3.0% of body weight per day)",
+      "Value": "0.03",
+      "StaticDataVersionId": 4
+    }
+  ],
+  "Feeds": [
+    {
+      "Id": 1,
+      "Name": "Orchardgrass",
+      "CPPercent": "12.9",
+      "PhosphorousPercent": "0.21",
+      "PotassiumPercent": "1.87",
+      "FeedForageTypes": [
+        {
+          "Id": 1,
+          "Name": "Hay - perennials",
+          "FeedId": 4,
+          "StaticDataVersionId": 4
+        }
+      ],
```

Comment:
These collections should be removed for each record and go into their own table json data.   This has caused the creation of 5 Forage Feed Types, where should only be three.

Hay - perennials
Cereal
Silage

The Feed, should only have the single ID for the FeedForageType.